code/datums/ert/outsider.dm

```diff
@@ -10,9 +10,16 @@
 
 /datum/responseteam/iac
 	name = "Interstellar Aid Corps"
-	chance = 15
```

Comment:
I think the chance can be zero and it will never be picked, pretty sure the admin var is no longer needed

Reply:
Tried that before, they were still somehow selected.